src/plugins/ui_feature/reorganize_sheet.ts

```diff
@@ -0,0 +1,572 @@
+import { ClipboardHandler } from "../../clipboard_handlers/abstract_clipboard_handler";
+import { DEFAULT_CELL_HEIGHT } from "../../constants";
+import {
+  compareZoneByWidth,
+  getZoneArea,
+  mergeContiguousZones,
+  positionToZone,
+  range,
+  recomputeZones,
+  zoneToDimension,
+} from "../../helpers";
+import { getClipboardDataPositions } from "../../helpers/clipboard/clipboard_helpers";
+import {
+  CellValueType,
+  Command,
+  CommandResult,
+  Dimension,
+  Figure,
+  HeaderIndex,
+  Pixel,
+  Position,
+  UID,
+  Zone,
+} from "../../types/index";
+import { UIPlugin } from "../ui_plugin";
+
+const INSERT_BOTTOM_MARGIN = 2; // in number of rows
+
+//ADRM TODO: send to sheet: manage borders
```

Comment:
:eyes: borders... :roll_eyes:

Reply:
It was done (and tested), I just forgot the TODO